src/tree_sitter_parser.rs

```diff
@@ -379,6 +380,16 @@ pub fn from_language(language: guess::Language) -> TreeSitterConfig {
                 .unwrap(),
             }
         }
+        Scala => {
+            let language = unsafe { tree_sitter_scala() };
+            TreeSitterConfig {
+                name: "Rust",
```

Comment:
```suggestion
                name: "Scala",
```

You probably want to change this :)

Reply:
Oops. Thanks for catching it!